products/compute/api.yaml

```
@@ -935,6 +935,162 @@ objects:
         description: |
           How many seconds to wait for the backend before considering it a
           failed request. Default is 30 seconds. Valid range is [1, 86400].
+  - !ruby/object:Api::Resource
+    name: 'RegionBackendService'
+    kind: 'compute#backendService'
+    base_url: projects/{{project}}/regions/{{region}}/backendServices
+    collection_url_response: !ruby/object:Api::Resource::ResponseList
+      kind: 'compute#backendServicesList'
+      items: 'items'
+    has_self_link: true
+    description: |
+      A Region Backend Service defines a regionally-scoped group of virtual
+      machines that will serve traffic for load balancing.
+
+      ~> **Note**: Region backend services can only be used when using internal load balancing.
```

```suggestion
      Region backend services can only be used when using internal load balancing.
```

The warning box syntax will appear as symbols in most markdown processors, so let's remove it.